src/Symfony/Bundle/DebugBundle/Resources/config/services.xml

```diff
@@ -36,11 +36,6 @@
             <argument>null</argument>
             <argument>%kernel.charset%</argument>
             <argument>0</argument> <!-- flags -->
-            <call method="setDisplayOptions">
-                <argument type="collection">
-                    <argument key="fileLinkFormat" type="service" id="debug.file_link_formatter"></argument>
```

Comment:
much simpler would be to use ``on-invalid="ignore"`` instead of changing it in a compiler pass